setup.py

```diff
@@ -49,6 +49,8 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Topic :: Software Development :: Quality Assurance",
         "Topic :: Software Development :: Testing",
+        "Typing :: Typed",
```

Comment:
Seems like you should also add `Topic :: Text Processing :: Markup :: LaTeX`
https://pypi.org/classifiers/